corehq/apps/notifications/views.py

```diff
@@ -48,7 +48,7 @@ def get_notifications(self, in_data):
             plan_tier = 'basic'
 
         def should_hide_feature_notifs(domain, plan):
-            groups = Group.get_case_sharing_groups(domain)
+            groups = Group.get_case_sharing_groups(domain, wrap=False)
```

Comment:
Unrelated to changes on this PR, but do you find `wrap` to be as confusing as I do in this context?